### DISCUSSION TITLE
Feature Request: Add a drop down menu

### DISCUSSION THREAD

**Discussion by chris-short** [adityatelange/hugo-PaperMod#379] [Ideas]
I’m not sure the level of effort but it’d be dope to have a drop down menu of user selected urls to help folks navigate important areas of a website.

**Comment by adityatelange** [comment]
I can understand the need of a drop-down menu, or maybe a hamburger menu for mobile devices. But it diverges from the design goal which paper had and on which papermod follows on. 
With the waves of features, I would not like to take out the simplicity the theme has, the lightness and how performance orientated it is.

By adding a drop-down menu will be an invitation to more features like nested menus, transitions/animations and more. 
As a result I'm not really inclining towards this feature, coz I don't know in which direction this could possibly go.

**Reply by seanmrnda** [reply]
Making the responsive hamburger menu optional is not a bad idea. It'll be up to the users of the theme to whichever they prefer.
